teensy40_brushed_v2/src/main_ros.cpp

```diff
@@ -0,0 +1,469 @@
+    float cur3_voltage = mot3._current_sensor->get_raw_voltage() - 1.5;
+    float smoothed_cur1_voltage = moving_average(cur1_voltage, cur1_voltage_buffer, MOVING_AVERAGE_SIZE, &cur1_voltage_buffer_idx);
+    float smoothed_cur2_voltage = moving_average(cur2_voltage, cur2_voltage_buffer, MOVING_AVERAGE_SIZE, &cur2_voltage_buffer_idx);
+    float smoothed_cur3_voltage = moving_average(cur3_voltage, cur3_voltage_buffer, MOVING_AVERAGE_SIZE, &cur3_voltage_buffer_idx);
+
+    // HWSERIAL.printf("cur1_current: %8.4f, cur2_current: %8.4f, cur3_current: %8.4f ",
+    //                  cur1_current, cur2_current, cur3_current);
+    // HWSERIAL.printf("cur1_voltage: %8.4f, cur2_voltage: %8.4f, cur3_voltage: %8.4f, ",
+    //                 smoothed_cur1_voltage, smoothed_cur2_voltage, smoothed_cur3_voltage);
+
+    // TEST ENCODER ------------------------------------------------------------------------------
+
+    // Channel 3 is conflicted, unused.
+    mot1._encoder->poll_encoder_angle();
+    mot2._encoder->poll_encoder_angle();
+    // mot3._encoder->poll_encoder_angle();
+
+    // Read encoder values.
+    float enc1_quad_enc_pos = mot1._encoder->_encoder->read();
+    float enc1_angle_single = mot1._encoder->get_angle_single();
+    float enc1_angle_es = mot1.get_current_angle_es();
+    float enc1_angle_ps = mot1.get_current_angle_ps();
+    float enc1_setpoint = mot1.get_target_angle_ps();
+
+    float enc2_quad_enc_pos = mot2._encoder->_encoder->read();
+    float enc2_angle_single = mot2._encoder->get_angle_single();
+    float enc2_angle_es = mot2.get_current_angle_es();
+    float enc2_angle_ps = mot2.get_current_angle_ps();
+    float enc2_setpoint = mot2.get_target_angle_ps();
+
+    // float enc2_quad_enc_pos = mot2._encoder->_encoder->read();
+    // float enc2_angle_single = mot2._encoder->get_angle_single();
+    // float enc2_angle_multi = mot2._encoder->get_angle_multi();
+    // float enc3_quad_enc_pos = mot3._encoder->_encoder->read();
+    // float enc3_angle_single = mot3._encoder->get_angle_single();
+    // float enc3_angle_multi = mot3._encoder->get_angle_multi();
+
+    HWSERIAL.printf("enc1_quad_enc_pos: %8.4f, enc1_angle_single: %8.4f, enc1_angle_es: %8.4f, enc1_angle_ps: %8.4f, enc1_setpoint: %8.4f, ",
+                    enc1_quad_enc_pos, enc1_angle_single, enc1_angle_es, enc1_angle_ps, enc1_setpoint);
+    HWSERIAL.println();
+    HWSERIAL.printf("enc2_quad_enc_pos: %8.4f, enc2_angle_single: %8.4f, enc2_angle_es: %8.4f, enc2_angle_ps: %8.4f, enc2_setpoint: %8.4f, ",
+                    enc2_quad_enc_pos, enc2_angle_single, enc2_angle_es, enc2_angle_ps, enc2_setpoint);
+
+    // float enc1_rev = mot1._encoder->_encoder->getRevolution();
+    // float enc1_hold_rev = mot1._encoder->_encoder->getHoldRevolution();
+    // HWSERIAL.printf("enc1_rev: %8.4f, enc1_hold_rev: %8.4f, ", enc1_rev, enc1_hold_rev);
+    HWSERIAL.println();
+}
+
+void brushed_board_ros_loop()
+{
+#if DEBUG_PRINT == 1
```

Comment:
There are a few of these #if <SOME_MACRO> == 1. I would suggest
```suggestion
#ifdef DEBUG_PRINT
```